packages/jellyfish-api-core/__tests__/category/spv/listHtlcsOutputs.test.ts

```diff
@@ -0,0 +1,76 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+
+describe('Spv', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.call('spv_fundaddress', [await container.call('spv_getnewaddress')]) // Funds 1 BTC
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listHtlcOutputs', async () => {
```

Comment:
Add tests so that you cover following

- multiple htlcs, all queried with `listHtlcOutputs` 
- htlc outputs queried with `listHtlcOutputs(htlc.address)`